src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -136,16 +140,26 @@ export default defineComponent<DashboardBoardListProps>({
     setup(props) {
         const state = reactive({
             thisPage: 1,
-            dashboardScopeType: computed(() => DASHBOARD_SCOPE[props.scopeType]),
-            dashboardScopeTypeForView: computed(() => (props.scopeType === DASHBOARD_SCOPE.DOMAIN ? 'Workspace' : 'Project')),
+            dashboardScopeKey: computed(() => (props.scopeType === DASHBOARD_SCOPE.DOMAIN ? DOMAIN_SCOPE_KEY : PROJECT_SCOPE_KEY)),
+            projectItems: computed(() => store.state.reference.project.items),
```

Comment:
suggestion: Please use getters except when neccessary! ([Why should we use getters?](https://www.notion.so/spaceone-io/Reference-Store-getters-key-10ce040c25894624a62ad05d51c1d1cc))

```suggestion
            projects: computed<ProjectReferenceMap>(() => store.getters['reference/projectItems']),
```

Reply:
Thanks!!